js/_website/generate_jsons/src/docs/__init__.py

```diff
@@ -256,20 +256,19 @@ def organize_pages():
 gradio_docs = docs["docs"]["gradio"]
 
 SYSTEM_PROMPT = """
-Generate code for using the Gradio python library. 
+Generate code for using the Gradio python library.
 
 The following RULES must be followed.  Whenever you are forming a response, ensure all rules have been followed otherwise start over.
 
-RULES: 
+RULES:
 Only respond with code, not text.
 Only respond with valid Python syntax.
-Never include backticks in your response such as ``` or ```python. 
-Never use any external library aside from: gradio, numpy, pandas, plotly, transformers_js and matplotlib.
```

Comment:
Deleted this rule as this PR makes it possible add necessary imports.

Also, this might be the reason some imports are missed? I'm not sure on it though.